synapse/config/emailconfig.py

```diff
@@ -250,18 +252,24 @@ def default_config(self, config_dir_path, server_name, **kwargs):
         #   #
         #   # If this option is set to false and SMTP options have not been
         #   # configured, resetting user passwords via email will be disabled
+        #   #
         #   #trust_identity_server_for_password_resets: false
         #
         #   # Configure the time that a validation email or text message code
         #   # will expire after sending
         #   #
         #   # This is currently used for password resets
+        #   #
         #   #validation_token_lifetime: 1h
         #
         #   # Template directory. All template files should be stored within this
         #   # directory
         #   #
-        #   #template_dir: res/templates
+        #   # If not set, the default location is `(location of Synapse's
+        #   # virtualenv)/res/templates`. Using an absolute path is recommended
```

Comment:
this implies it's in a `res` directory within the virtualenv, which isn't quite right (it's in `env/lib/python3.6/site-packages/synapse/res/templates` for me here).

Also... it's *possible* (though not recommended) to run synapse from a working copy rather than a pip install, in which case it wouldn't be in the virtualenv at all.

I think it might just be best to say "if not set, default templates from within the synapse package will be used".